frontend/src/old-pages/Clusters/Cost.tsx

```diff
@@ -0,0 +1,293 @@
+              direction="horizontal"
+              size="xs"
+            >
+              <DateRangePicker
+              onChange={({ detail }) => {fetchGraphData(detail.value); setDateValue(detail.value); }}
+                value={dateValue}
+                relativeOptions={[
+                  {
+                    key: "previous-30-days",
+                    amount: 30,
+                    unit: "day",
+                    type: "relative"
+                  },
+                  {
+                    key: "previous-14-days",
+                    amount: 14,
+                    unit: "day",
+                    type: "relative"
+                  },
+                  {
+                    key: "previous-7-days",
+                    amount: 7,
+                    unit: "day",
+                    type: "relative"
+                  },
+                  {
+                    key: "previous-1-day",
+                    amount: 1,
+                    unit: "day",
+                    type: "relative"
+                  }
+                ]}
+                i18nStrings={{
+                  todayAriaLabel: "Today",
+                  nextMonthAriaLabel: "Next month",
+                  previousMonthAriaLabel: "Previous month",
+                  customRelativeRangeDurationLabel: "Duration",
+                  customRelativeRangeDurationPlaceholder:
+                    "Enter duration",
+                  customRelativeRangeOptionLabel: "Custom range",
+                  customRelativeRangeOptionDescription:
+                    "Set a custom range of days in the past. Use Days as Unit of Time.",
+                  customRelativeRangeUnitLabel: "Unit of time",
+                  formatRelativeRange: e => {
+                    const t =
+                      1 === e.amount ? e.unit : `${e.unit}s`;
+                    return `Last ${e.amount} ${t}`;
+                  },
+                  formatUnit: (e, t) => (1 === t ? e : `${e}s`),
+                  dateTimeConstraintText:
+                    "Select any Range of days. Use 24 hour format.",
+                  relativeModeTitle: "Relative range",
+                  absoluteModeTitle: "Absolute range",
+                  relativeRangeSelectionHeading: "Choose a range",
+                  startDateLabel: "Start date",
+                  endDateLabel: "End date",
+                  startTimeLabel: "Start time",
+                  endTimeLabel: "End time",
+                  clearButtonLabel: "Clear and dismiss",
+                  cancelButtonLabel: "Cancel",
+                  applyButtonLabel: "Apply"
+                }}
+                placeholder="Filter by a date range" />
+        <Button disabled={tag_active} onClick={activate_tags} variant="primary">Activate Tags</Button> 
+        <HelpTooltip> 
+        The NoInstanceType category includes miscellaneous costs such as EBS, read more about <a href="https://aws.amazon.com/ebs/pricing/" target="-blank"> EBS Pricing</a>.
```

Comment:
This should be internationalized -- here's an example of where this was done previously: https://github.com/aws-samples/pcluster-manager/pull/218